packages/shared/comms/v3/BFFConnection.ts

```diff
@@ -176,9 +196,6 @@ export class BFFConnection {
     const auth = loadService(port, BffAuthenticationServiceDefinition)
 
     const getChallengeResponse = await auth.getChallenge({ address })
-    if (getChallengeResponse.alreadyConnected) {
```

Comment:
@menduz this was causing

```
Sep 12 12:23:05 peer-ec2 explorer-bff[1549]: 2022-09-12T12:23:05.760Z [ERROR] (rpc-server): Error handling remote request#011{
"message":"Module CommsService is not available for port kernel (4051))",
"name":"Error",
"stack":"Error: Module CommsService is not available for port kernel (4051))
    at Object.loadModule (/app/node_modules/@dcl/rpc/dist/server.js:82:19)
    at handleRequestModule (/app/node_modules/@dcl/rpc/dist/server.js:126:37)
    at handleMessage (/app/node_modules/@dcl/rpc/dist/server.js:263:19)
    at /app/node_modules/@dcl/rpc/dist/server.js:299:35
    at /app/node_modules/mitt/dist/mitt.js:1:255
    at Array.map (<anonymous>)
    at Object.emit (/app/node_modules/mitt/dist/mitt.js:1:239)
    at WebSocket.<anonymous> (/app/node_modules/@dcl/rpc/dist/transports/WebSocket.js:54:20)
    at WebSocket.onMessage (/app/node_modules/ws/lib/event-target.js:199:18)
    at WebSocket.emit (node:events:513:28)"}
 ```

Reply:
I'd track an event here! it means that we are kicking another session

Reply:
you mean a metric? can you give me an example? I'll merge this as it is, because I want to clear the noise, but I'll add the metric in another PR

Reply:
look for function trackEvent()